src/utils/convert.util.ts

```diff
@@ -71,11 +91,11 @@ export function hsv2rgb({ h, s, v }: Color["hsv"]): Color["rgb"] {
   const g = Math.round([t, v, v, q, p, p][index] * 255);
   const b = Math.round([p, p, t, v, v, q][index] * 255);
 
-  return { r, g, b };
+  return { r, g, b, a };
 }
 
-export function rgb2hex({ r, g, b }: Color["rgb"]): Color["hex"] {
-  const hex = [r, g, b].map((v) => v.toString(16).padStart(2, "0")).join("");
+export function rgb2hex({ r, g, b, a }: Color["rgb"]): Color["hex"] {
+  const hex = [r, g, b, a].map((v) => (v !== undefined ? v.toString(16).padStart(2, "0") : "")).join("");
```

Comment:
```ts
const hex = [r, g, b, a].map((v) => (v ? v.toString(16).padStart(2, "0") : "")).join("");
```

Reply:
This change would cause any value equal to zero to emit an empty string, since `v ? (...) : (...)` implicitly casts to a boolean and zero is a falsy value.